RoadMap_SwiftUI/RoadMap_SwiftUI/Helper/ShakeEffect.swift

```diff
@@ -0,0 +1,22 @@
+// ShakeEffect.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Эффект тряски
+struct ShakeEffect: GeometryEffect {
+    // MARK: - Public properties
+    var travelDistance: CGFloat = 6
+    var numOfShake: CGFloat = 4
+    
+    // MARK: - Private properties
+    var animatableData: CGFloat
```

Comment:
```suggestion
    // MARK: - Private properties
    
    var animatableData: CGFloat
```
не приватное свойство